src/utilities/xml/XMLValidator.hpp

```diff
@@ -52,7 +52,7 @@ class UTILITIES_API XMLValidator
   XMLValidator& operator=(const XMLValidator& other) = default;
   XMLValidator& operator=(XMLValidator&& other) noexcept = default;
 
-  static XMLValidator gbxmlValidator();
+  static XMLValidator gbxmlValidator(const VersionString& schemaVersion = VersionString(7, 3));
```

Comment:
Make it clearer that this is 7.03 schema, and reserve use for supporting more schemas.